docs/dev-guide/contributing/branching.rst

```diff
@@ -6,38 +6,35 @@ the platform, and then each supported content family (like "rpm" and "puppet")
 has its own repository.
 
 Pulp uses a version scheme x.y.z. Pulp's branching strategy is designed for
-bugfix development for older ``x.y`` release streams without interfering with
-development or contribution of new features to a future, unreleased ``x`` or
-``x.y`` release. This strategy encourages a clear separation of bugfixes and
-features as encouraged by `Semantic Versioning <http://semver.org/>`_.
+bugfix development on the ``master`` branch with cherry-picking fixes into
++``x.y`` streams as necessary. This version scheme is based
++on the `Semantic Versioning <http://semver.org/>`_ strategy.
```

Comment:
++ is a copy-paste artifact, the second + is a part of the actual document

Reply:
Yeah, ctl-s failure.  I guess we had a race condition in you reviewing and me pushing, but that's fixed now